docs/instructions.md

```diff
@@ -75,10 +74,44 @@ After following the steps in [Quick Start](#quick-start-app) above:
 
 ### App Settings ###
 
+#### iOS
+
+To change your general settings for the Whisper app, click the Settings button on the start screen. You will be automatically redirected to the Whisper section of your device's Settings app, which contains two lists of options. The first, "Allow Whisper to Access," lets you control how the app interacts with other features on your device. The second, "Whisper Settings," lets you control how the app itself behaves. 
+
+**Allow Whisper to Access** has four sections:
+
+* **Bluetooth** — allowing Whisper access to Bluetooth is what lets you start and join in-person conversations automatically when you tap Whisper or Listen. We highly recommend allowing Bluetooth access!
+* **Siri & Search** — tapping this option brings you to a menu where you can refine how much access Apple has to how you use Whisper, as well as whether you can use the built-in voice command functions to find and open Whisper
+* **Notifications** — tapping this option brings you to a menu where you can change what notifications (if any) the Whisper app will give you, the style of those notifications, and whether you'll see whispers on your lock screen
+  * This is where you choose whether the app alerts you with a sound when a new whisper comes in; to choose a particular alert sound, go to "Alert Sound" in Whisper Settings
+* **Background App Refresh** — this option is permanently set to on [explanation from Dad]
+
+ **Whisper Settings** has five sections:
+
+* **Remember name from last session** — turning this option on means you don't have to type in your name every time you join a conversation in the app. If multiple people use Whisper on the same device, it may make more sense to turn this option off so no one accidentally joins a conversation under the wrong name.
+* **Require Bluetooth listeners to pair** — [[I don't know what this means]]
```

Comment:
When this is off, the Whisperer will automatically connect to every Bluetooth listener that tries to join the conversation.  Bill was concerned that, once Whisper gets "popular" (ha ha), he might be in a restaurant or some public location and people would snoop on his conversation using Whisper.  So at his request I added this preference.  If it's turned on, then when a listener device wants to pair with a particular whisperer device over bluetooth for the very first time, then both the whisperer and the listener are required to "pair" with each other (same as when you bluetooth pair a headset or speaker to your phone).  SInce we are talking an iOS to iOS pairing, it doesn't happen automatically (the way it does with your speaker): both the whisperer and the listener are shown a pairing screen with a number on it, and each has to confirm they want to pair with the other person who has that number on their screen.  (This is how, for example, phones pair with cars - by code confirmation.). Pairing is a 1-time only thing: once it has been set up between two devices it never has to happen again (at least not until their next major OS upgrade...).

If someone is very security conscious (think Joe Cotsirilos or his mom ;), they turn this setting on to make sure no one can ever eavesdrop on them that they haven't approved.  Bill has never used it, to my knowledge.

Reply:
This one is going to take some time for me to word properly, just fyi